tests/e2e/esm/index.test.ts

```diff
@@ -1108,4 +1109,60 @@ describe('e2e esm tests', async () => {
       }
     });
   });
+
+  // Skipping due to inconsistent results with spun up CU
+  describe.skip('AOProcess', async () => {
+    it('catches errors that appear on msg.Error', async () => {
+      const processId = 'KF4wSJa5no2yyDjrDE3hxd5sJoGhLh0UEj8a6Rz3aX8'; // testnet airdrop
+      const process = new AOProcess({
+        processId,
+        ao: aoClient,
+      });
+
+      const error = await process
+        .send({
+          signer: signers[1],
+          tags: [
+            {
+              name: 'Action',
+              value: 'Register',
+            },
+          ],
+        })
+        .catch((e) => e);
+
+      assert.ok(error);
+      assert.ok(error instanceof WriteInteractionError);
+      assert.equal(
+        error.message.split('(')[0].trim(),
+        'Recipient not found for wallet address',
+      );
+    });
```

Comment:
these tests are actually writing message to our airdrop process - we likely don't want to do that